src/timers.cc

```diff
@@ -127,7 +127,7 @@ double ProcessCPUUsage() {
   DiagnoseAndExit("GetProccessTimes() failed");
 #elif defined(BENCHMARK_OS_QURT)
   return static_cast<double>(
-             qurt_timer_timetick_to_us(qurt_timer_get_ticks())) *
+             qurt_timer_timetick_to_us(qurt_sysclock_get_hw_ticks())) *
```

Comment:
Can you please add in-code comment why this uses `qurt_sysclock_get_hw_ticks` instead of `qurt_timer_get_ticks`?

Reply:
Done